internal/exec/stages/disks/filesystems.go

```diff
@@ -190,17 +191,22 @@ func (s stage) createFilesystem(fs types.Filesystem) error {
 		if fs.Label != nil {
 			args = append(args, "-n", *fs.Label)
 		}
+	case "none":
+		// User specifies this format to skip the creation of a filesystem on a block device.
+		// For more information https://github.com/coreos/ignition/issues/1172
+		noneFileSystemFormatFlag = true
 	default:
 		return fmt.Errorf("unsupported filesystem format: %q", *fs.Format)
 	}
-
-	args = append(args, devAlias)
-	if _, err := s.Logger.LogCmd(
-		exec.Command(mkfs, args...),
-		"creating %q filesystem on %q",
-		*fs.Format, devAlias,
-	); err != nil {
-		return fmt.Errorf("mkfs failed: %v", err)
+	if !noneFileSystemFormatFlag {
```

Comment:
Hmm, feels like we could just drop this var and here check instead `if *fs.Format == "none" {`.

Reply:
You mean `*fs.Format !=  "none"`, right?

Reply:
Yes, exactly!